api/backend.go

```diff
@@ -298,21 +311,6 @@ func (b *StatusBackend) registerHandlers() error {
 		return b.AccountManager().Accounts()
 	})
 
-	rpcClient.RegisterHandler(params.SendTransactionMethodName, func(ctx context.Context, rpcParams ...interface{}) (interface{}, error) {
-		txArgs, err := transactions.RPCCalltoSendTxArgs(rpcParams...)
-		if err != nil {
-			return nil, err
-		}
-
-		hash, err := b.SendTransaction(ctx, txArgs)
-		if err != nil {
-			return nil, err
-		}
-
-		return hash.Hex(), err
-	})
-
-	rpcClient.RegisterHandler(params.PersonalSignMethodName, b.personalAPI.Sign)
 	rpcClient.RegisterHandler(params.PersonalRecoverMethodName, b.personalAPI.Recover)
```

Comment:
@divan @andrey Do we also need to do the same for personalRecover? If we do, maybe we should split this PR up, as its getting pretty massive and taking a long time. We could do separate PRs for `sendTransaction`, `signMessage` and `ecRecover`

Reply:
From status-react perspective, it would be better to have all these calls changed at once I believe. On status-go side, we can have many PRs merged to some feature branch and then the feature branch would be merged to `develop` without detailed review.